### ISSUE TITLE
Heavy network bandwidth usage

### ISSUE THREAD

**Issue by joshieDo** [paradigmxyz/reth#709]
### Describe the bug

Heavy network bandwidth usage ~~with 0 good peers.~~

edit: And apparently, it's accompanied by a high CPU usage.

### setup
Running `reth` from a minimal ubuntu server a `LXC` container. This is connected to another `LXC` container for networking. Which in turn is connected to a vpn.

I run this setup `node(lxc) -> network(lxc) -> vpn` with other nodes with no networking issues afaik (bitcoin, monero, goerli).

### issue
Left it yesterday for an hour, and I found 0 good peers. Furthermore, the network usage seemed too excessive.

During periods of time it just maxs it completely, and I even lose `ssh` connection to the machine. This can either be during a couple seconds, to couple minutes. I only have the `reth` process running, and killing it (through proxmox UI) completely solves it.

![image](https://user-images.githubusercontent.com/93316087/210499097-08d9957b-9919-473c-9dee-8eccda149bcf.png)

![image](https://user-images.githubusercontent.com/93316087/210503189-6738ed40-ba73-4be9-8d5b-db46c2098f5d.png)

![image](https://user-images.githubusercontent.com/93316087/210503427-e67b7da4-ecfe-4cc1-818b-06b9bf768ef5.png)

Querying `localhost:3000`, the only rising value is: `reth_p2pstream_disconnected_errors 191`

If there's anything you want me run lmk. Currently running the node like:

```
RUST_LOG=info,sync::stages=trace,downloaders=trace ./reth node -v --db $DB --metrics 127.0.0.1:3001
```

### Platform(s)

Linux (x86)

### Code of Conduct

- [X] I agree to follow the Code of Conduct

**Comment by mattsse** [comment]
> with 0 good peers

what exactly do you mean here? I doubt this traffic is just from _trying_ to establish connections

**Comment by joshieDo** [comment]
I have 0 headers 0 bodies, and to my knowledge haven't been able to find 1 good peer

**Comment by mattsse** [comment]
these spikes look like lookup interval, it also makes sense why there are large spikes at the beginning and then it evens out.

this is due to the recursive lookup step, I'll come up with a fix that does not result in these heavy spikes

**Comment by gakonst** [comment]
Closed by #731 #733
